kinto/core/logs.py

```diff
@@ -10,6 +10,19 @@
 logger = structlog.get_logger()
 
 
+def reset_logger():
+    """Hack to work around https://github.com/hynek/structlog/issues/71.
+
+    This clears a magic field in the logger so that it will regenerate
+    the right kind of logger on its next use.
+
+    Do this when you call structlog.configure(). Otherwise, some parts
```

Comment:
OK. Since we call `configure` in `initialization.py:setup_logging`, I've just moved this code there in 9f83849eb4067c1514abfc91c1afea5306aad7cc.